pytorch_lightning/core/optimizer.py

```diff
@@ -255,9 +255,8 @@ def _configure_schedulers(
     """Convert each scheduler into dict structure with relevant information."""
     lr_schedulers = []
     default_config = _get_default_scheduler_config()
-    # TODO: move is_manual_optimization check out of for loop
-    for scheduler in schedulers:
-        if is_manual_optimization:
+    if is_manual_optimization:
```

Comment:
i'd add 2 helper methods for the validation, one for automatic opt and one for manual opt, since they have different conditions/checks